html/modules/custom/reliefweb_entities/reliefweb_entities.module

```diff
@@ -1326,6 +1326,33 @@ function reliefweb_entities_node_update(EntityInterface $entity) {
   reliefweb_entities_notify_webhooks('reliefweb:entity_updated', $payload);
 }
 
+/**
+ * Implements hook_ENTITY_TYPE_update().
+ *
+ * Notify webhooks about the entity update.
+ */
+function reliefweb_entities_taxonomy_term_update(EntityInterface $entity) {
+  /** @var \Drupal\taxonomy\TermInterface $entity */
+
+  // Only for country and source.
+  if (!in_array($entity->bundle(), ['country', 'source'], TRUE)) {
+    return;
+  }
+
+  // Ignore if the entity is not published.
+  if (!$entity->isPublished()) {
```

Comment:
I would remove that so that we can also send an update when sources are archived for example.